frontend/src/pages/Landing/index.js

```diff
@@ -164,6 +169,7 @@ function Landing() {
     return <div>Loading...</div>;
   }
 
+  const message = history.location.state && history.location.state.message;
```

Comment:
We may want to clear out the message after it is loaded, simliar to the [activity report](https://github.com/HHS/Head-Start-TTADP/blob/main/frontend/src/pages/ActivityReport/index.js#L81). If you refresh the page after submitting the report you see the message again. I would lean toward clearing the message but could see the case for keeping the message on refresh.

Reply:
Clearing the message makes sense.